src/database.ts

```diff
@@ -849,15 +1024,18 @@ class Database extends ServiceObject {
         gaxOpts,
       },
       // tslint:disable-next-line only-arrow-functions
-      function(err, sessions) {
+      function(
+        err: ServiceError | null,
+        sessions: spanner_client.spanner.v1.Session[]
+      ) {
         if (sessions) {
           arguments[1] = sessions.map(metadata => {
             const session = self.session(metadata.name);
             session.metadata = metadata;
             return session;
           });
         }
-        callback.apply(null, arguments);
+        callback.apply(null, arguments as Any);
```

Comment:
```suggestion
        callback!.apply(null, arguments as Any);
```
as "only call `callback` if it is provided" otherwise it will be picked up by `PromisifyAll`